cmd/juju/application/bundle_test.go

```diff
@@ -144,33 +129,27 @@ func (s *BundleDeploySuite) setupCharmMaybeForce(c *gc.C, url, name string, abas
 	}
 
 	var chDir charm.Charm
-	chDir, err = charm.ReadCharmDir(testcharms.RepoWithSeries(aseries).CharmDirPath(name))
+	chDir, err := charm.ReadCharmDir(testcharms.RepoWithSeries("bionic").CharmDirPath(name))
```

Comment:
`bionic` is just the folder name in the charm repot folder right?

Reply:
Yep

Reply:
It turns out almost there is enough overlap between test charms in these dirs that we can just used 'bionic' and tests pass